src/console_title_test.go

```diff
@@ -49,6 +59,8 @@ func TestGetConsoleTitle(t *testing.T) {
 		env.On("isRunningAsRoot", nil).Return(tc.Root)
 		env.On("getShellName", nil).Return(tc.ShellName)
 		env.On("getenv", "USERDOMAIN").Return("MyCompany")
+		env.On("getCurrentUser", nil).Return("MyUser")
+		env.On("getHostName", nil).Return("MyHost", nil)
```

Comment:
Can we create a test where we expect a host name but the command errors? Check if the template works.

Reply:
I wrote the test and confirmed that template works without the Host value, but it prints "&lt;no value&gt;" instead.
So I changed the code to put an empty string on Host if the getHostName returns an error.